src/main/java/me/totalfreedom/totalfreedommod/command/Command_mute.java

```diff
@@ -17,7 +17,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.ADMIN, source = SourceType.BOTH)
-@CommandParameters(description = "Mutes a player with brute force.", usage = "/<command> <[-s | -q] <player> [reason] | list | purge | all>", aliases = "stfu")
+@CommandParameters(description = "Mutes a player with brute force.", usage = "/<command> <[-s | -q] <player> [reason] | list | purge | -a>", aliases = "stfu")
```

Comment:
Can you re-word the description. It's just something left over from when the command was primarily used as /stfu which tbf is an alias we will probably look to remove in the near future...

Reply:
nah i think /stfu and /gtfo fits the charm of the server, also /stfu is still used over /mute to this day

Reply:
fixed